### PR TITLE
TN-2849 RP, QB and Questionnaire for post-intervention.

### PR DESCRIPTION
**Pull request by pbugni**
Initial take at the RP, QB and Questionnaire needed for post-intervention.

The questionnaire doesn't yet have any of the question structure - @mcjustin is this something you'd be willing to fill in?

### CODE REVIEW

File: portal/config/eproms/Questionnaire.json [hunk 1]
```diff
@@ -7843,6 +7843,23 @@
           "valueBoolean": true
         }
       ]
+    },
+    {
+      "identifier": [
+        {
+          "system": "http://us.truenth.org/questionnaire",
+          "value": "ironman_post_intervention"
```
Comment (mcjustin) [inline portal/config/eproms/Questionnaire.json:7851]: You might abbreviate intervention, perhaps "tx"?

File: portal/config/eproms/QuestionnaireBank.json [hunk 2]
```diff
@@ -769,6 +769,58 @@
           "rank": 0
         }
       ]
+    },
+    {
+      "resourceType": "QuestionnaireBank",
+      "classification": "baseline",
+      "name": "empro_post_intervention_baseline",
+      "start": "{\"days\": 0}",
```
Comment (mcjustin) [inline portal/config/eproms/QuestionnaireBank.json:777]: How do you indicate what event these are anchored on? eg for this case, it's the creation of a trigger (vs ironman main study using consent date)

File: portal/config/eproms/Questionnaire.json (hunk 1 above)
Comment (pbugni) [reply portal/config/eproms/Questionnaire.json]: Good idea.  Now using `ironman_ss_post_tx`.

File: portal/config/eproms/QuestionnaireBank.json (hunk 2 above)
Comment (pbugni) [reply portal/config/eproms/QuestionnaireBank.json:777]: it's beyond the scope of the QB definition, for now.  handled via `trigger_states`.

File: portal/config/eproms/Questionnaire.json [hunk 3]
```diff
@@ -7843,6 +7843,17 @@
...
+    {
+      "identifier": [
+        {
+          "system": "http://us.truenth.org/questionnaire",
+          "value": "ironman_ss_post_tx"
+        }
+      ],
+      "status": "draft",
+      "resourceType": "Questionnaire",
+      "item": []
```
Comment (mcjustin) [inline portal/config/eproms/Questionnaire.json:7856]: @pbugni  am I correct in recalling that I should populate this?

Comment (mcjustin) [reply portal/config/eproms/Questionnaire.json]: Done @pbugni

Comment (pbugni) [reply portal/config/eproms/Questionnaire.json]: looks great, thank you @mcjustin

**Review by mcjustin — APPROVED** [review]

**Pull request merged**